### PR TITLE
[Term Entry] PHP Date function date_sub()

### PR DESCRIPTION
**Pull request by EspressoCat**
### Description

Added date_sub() term for PHP
Closes issue #2553 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR

### CODE REVIEW

File: content/php/concepts/date-functions/terms/date-sub/date-sub.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Syntax
+
+```pseudo
+date_sub($object, $interval)
+```
+
+The `date_sub()` function has two required parameters:
+
+- `$object`: This parameter specifies the DateTime object returned by date_create().
```
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-sub/date-sub.md:25]:
```suggestion
- `$object`: This parameter specifies the `DateTime` object returned by `date_create()`.
```

File: content/php/concepts/date-functions/terms/date-sub/date-sub.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- `$interval`: This parameter specifies the DateInterval object which we want to subtract.
```
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-sub/date-sub.md:26]:
```suggestion
- `$interval`: This parameter specifies the `DateInterval` object to be subtracted.
```

Comment (caupolicandiaz) [reply content/php/concepts/date-functions/terms/date-sub/date-sub.md:26]: we avoid the use of first and second person pronouns (I, you, we) in docs

File: content/php/concepts/date-functions/terms/date-sub/date-sub.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+The `date_sub()` function returns a DateTime object after subtracting the interval.
```
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-sub/date-sub.md:28]:
```suggestion
The `date_sub()` function returns a `DateTime` object after subtracting the interval.
```

File: content/php/concepts/date-functions/terms/date-sub/date-sub.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## Example
+
+The following example uses the `date_sub()` function to subtract 30 days from the 3 March, 2023:
```
Comment (caupolicandiaz) [inline content/php/concepts/date-functions/terms/date-sub/date-sub.md:32]:
```suggestion
The following example uses the `date_sub()` function to subtract 30 days from the passed date:
```

File: content/php/concepts/date-functions/terms/date-sub/date-sub.md (hunk 1 above)
Comment (caupolicandiaz) [reply content/php/concepts/date-functions/terms/date-sub/date-sub.md:25]: generally we use backticks on specific terms, functions, symbols specific to the code shown or language. If it's a generic term such as arrays or classes I would omit the backticks.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@EspressoCat I've completed my initial review. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by EspressoCat** [comment]
Thanks for the feedback @caupolicandiaz I made the requested changes.

**Review by caupolicandiaz — APPROVED** [review]
@EspressoCat I just reverted the changes on array-fill, they shouldn't be part of this PR. Looks good for a second review.

**Review by SSwiniarski — APPROVED** [review]
@EspressoCat Looks good to me.

**Comment by SSwiniarski** [comment]
@EspressoCat Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/date-functions/date-sub

image

**Pull request merged**